chapters/commonvalidity/resolve_image_common.txt

```diff
@@ -144,4 +144,20 @@ endif::VK_EXT_fragment_density_map[]
     If pname:dstImage is of type ename:VK_IMAGE_TYPE_1D or
     ename:VK_IMAGE_TYPE_2D, then for each element of pname:pRegions,
     pname:dstOffset.z must: be `0` and pname:extent.depth must: be `1`
+  * [[VUID-{refpage}-srcImage-00126]]
```

Comment:
VUID tags are assigned after an MR is accepted into main, so don't do this here and below - just make it a regular asciidoctor bullet list item e.g.

> * pname:srcImage ...